### PR TITLE
Add MSI Alpha 17 B5EEK support

### PR DESCRIPTION
**Pull request by teackot**

### CODE REVIEW

File: msi-ec.c [hunk 1]
```diff
@@ -994,6 +995,85 @@ static struct msi_ec_conf CONF11 __initdata = {
...
+		.address = 0xf4,
+		.modes = {
+			{ FM_AUTO_NAME,     0x0d },
+			{ FM_SILENT_NAME,   0x1d },
+			{ FM_ADVANCED_NAME, 0x8d },
+			MSI_EC_MODE_NULL
+		},
+	},
+	.cpu = {
+		.rt_temp_address       = 0x68, // needs testing
```
Comment (mutchiko) [inline msi-ec.c:1050]: tested, it is right

File: msi-ec.c [hunk 2]
```diff
@@ -994,6 +995,85 @@ static struct msi_ec_conf CONF11 __initdata = {
...
+		.rt_fan_speed_address  = 0x71, // needs testing
```
Comment (mutchiko) [inline msi-ec.c:1051]: tested, it is right

File: msi-ec.c [hunk 3]
```diff
@@ -994,6 +995,85 @@ static struct msi_ec_conf CONF11 __initdata = {
...
+		.rt_fan_speed_base_min = 0x19,
+		.rt_fan_speed_base_max = 0x37,
+		.bs_fan_speed_address  = MSI_EC_ADDR_UNSUPP,
+		.bs_fan_speed_base_min = 0x00,
+		.bs_fan_speed_base_max = 0x0f,
+	},
+	.gpu = {
+		.rt_temp_address      = MSI_EC_ADDR_UNKNOWN,
```
Comment (mutchiko) [inline msi-ec.c:1059]: it is at [0x80]

File: msi-ec.c [hunk 4]
```diff
@@ -994,6 +995,85 @@ static struct msi_ec_conf CONF11 __initdata = {
...
+		.rt_fan_speed_address = MSI_EC_ADDR_UNKNOWN,
```
Comment (mutchiko) [inline msi-ec.c:1060]: it's at [0x89]

**Comment by teackot** [comment]
Replaced by #122

**Pull request closed without merging**